dev/com.ibm.ws.logging/src/com/ibm/ws/logging/collector/LogFieldConstants.java

```diff
@@ -97,31 +97,33 @@
     public static final String REMOTEUSERID = "remoteUserID";
 
     //liberty_ffdc
-    public static final String DATEOFFIRSTOCCURENCE = "dateOfFirstOccurence";
-    public static final String COUNT = "count";
-    public static final String LABEL = "label";
+    //fields that contain 'XXXXXXX' are not expected to be used
+    public static final String DATEOFFIRSTOCCURENCE = "XXXXXXX_dateOfFirstOccurence";
+    public static final String COUNT = "XXXXXXX_count";
+    public static final String LABEL = "XXXXXXX_label";
     public static final String IBM_EXCEPTIONNAME = "ibm_exceptionName";
     public static final String EXCEPTIONNAME = "exceptionName";
     public static final String IBM_PROBEID = "ibm_probeID";
     public static final String PROBEID = "probeID";
-    public static final String SOURCEID = "sourceID";
+    public static final String SOURCEID = "XXXXXXX_sourceID";
```

Comment:
It seems like DATEOFFIRSTOCCURENCE, COUNT, LABEL, and SOURCEID are never used in output formats.  Please check to see if that's the case, and if so, we should remove them from these constants, as well as FFDCData / FFDCSource classes.